modules/apps/object/object-test/src/testFunctional/tests/tests/CreateObject.testcase

```diff
@@ -106,12 +106,33 @@ definition {
 	}
 
 	@description = "LPS-135549 - Verify it is possible to create a Long field"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test CanCreateLongField {
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object",
+			objectName = "CustomObject",
+			pluralLabelName = "Custom Objects");
+
+		Navigator.openURL();
+
+		ObjectAdmin.openObjectAdmin();
+
+		ObjectPortlet.selectCustomObject(name = "CustomObject");
+
+		ObjectAdmin.goToFieldsTab();
+
+		ObjectAdmin.addObjectFieldViaAPI(
+			fieldLabelName = "Custom Field",
+			fieldName = "customField",
+			fieldType = "Long",
+			isRequired = "false",
+			objectName = "CustomObject");
```

Comment:
Since for this test we need to validate that it is possible to create the field, you need to add the field via UI instead of the API.